File: URI/URI 1041 - Coordinates of a Point/URI 1041 - Coordinates of a Point.py
zero = 0

x , y =  list(map(float, input().strip().split(" "))) 

if x == zero and y == zero:
    print("Origem")
elif x == zero:
    print("Eixo Y")
elif y == zero:
    print("Eixo X")
elif x > zero and y > zero:
    print("Q1")
elif x < zero and y > zero:
    print("Q2")
elif x < zero and y < zero:
    print("Q3")
else:
    print("Q4")

File: URI/URI 1041 - Coordinates of a Point/URI 1041 - Coordinates of a Point.cpp
#include<bits/stdc++.h>
using namespace std;

int main(){

    double zero = 0;
    double x, y;
    cin >> x >> y;

    if (x == zero && y == zero)
        cout << "Origem" << endl;
    else if (x == zero)
        cout << "Eixo Y" <<endl;
    else if (y == zero)
        cout << "Eixo X" <<endl;
    else if (x > zero && y > zero)
        cout << "Q1" <<endl;
    else if (x < zero && y > zero)
        cout << "Q2" <<endl;
    else if (x < zero && y < zero)
        cout << "Q3" <<endl;
    else
        cout << "Q4" <<endl;

    return 0;
}